### PR TITLE
DC-4540: Updated scheduled jobs to use standard akka, removing depend…

### PR DESCRIPTION
**Pull request by ardwalker**
…ency on play-scheduler library

### CODE REVIEW

File: .bsp/sbt.json [hunk 1]
```diff
@@ -1 +1 @@
-{"name":"sbt","version":"1.6.2","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk1.8.0_161.jdk/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/kapilsanghvi/Library/Application Support/JetBrains/IdeaIC2021.1/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/Users/kapilsanghvi/Library/Application%20Support/JetBrains/IdeaIC2021.1/plugins/Scala/launcher/sbt-launch.jar"]}
\ No newline at end of file
+{"name":"sbt","version":"1.6.2","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/temurin-11.jdk/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/nix/store/rd34r9ksqsl3ikvhdsb747d1b9x4y2yq-sbt-1.6.2/share/sbt/bin/sbt-launch.jar","-Dsbt.script=/Users/andrew/.nix-profile/bin/sbt","xsbt.boot.Boot","-bsp"]}
```
Comment (kjainsc) [inline .bsp/sbt.json]: **.bsp** folder needs to be deleted I think,  we don't need it. I don't remember why I pushed it

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -20,31 +20,41 @@ import sbt._
...
+    "com.typesafe.akka"         %% "akka-actor-typed"                  % AkkaVersion,
+    "com.typesafe.akka"         %% "akka-protobuf-v3"                  % AkkaVersion,
+    "com.typesafe.akka"         %% "akka-stream"                       % AkkaVersion,
+    "com.typesafe.akka"         %% "akka-serialization-jackson"        % AkkaVersion,
+    "uk.gov.hmrc"               %% "bootstrap-backend-play-28"         % "6.4.0",
+    "uk.gov.hmrc"               %% "auth-client"                       % "5.7.0-play-28",
+    "uk.gov.hmrc"               %% "time"                              % "3.3.0",
+    "uk.gov.hmrc.mongo"         %% "hmrc-mongo-play-28"                % "0.70.0",
+    "uk.gov.hmrc.mongo"         %% "hmrc-mongo-work-item-repo-play-28" % "0.70.0",
+    "uk.gov.hmrc"               %% "play-scheduling-play-28"           % "8.1.0",
```
Comment (kjainsc) [inline project/AppDependencies.scala]: do we still need this dependency for **play-scheduling-play-28**

File: test/uk/gov/hmrc/ups/scheduled/UpdatedPrintSuppressionJobSpec.scala [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      val mlr = mock[MongoLockRepository]
+      val pp = mock[PreferencesProcessor]
+
+      when(mlr.takeLock(any[String], any[String], any[Duration])).thenReturn(Future.successful(true))
+      when(pp.run(any[HeaderCarrier])).thenReturn(Future.successful(TotalCounts(2, 2)))
+
+      system.actorOf(Props(new UpdatedPrintSuppressionJob(Configuration(config), mlr, pp)))
+
+      // TODO: Is there a better way to do this?
+      Thread.sleep(3000)
```
Comment (kjainsc) [inline test/uk/gov/hmrc/ups/scheduled/UpdatedPrintSuppressionJobSpec.scala]: could try with **Eventually** from scalatest?

File: project/AppDependencies.scala (hunk 2 above)
Comment (ardwalker) [reply project/AppDependencies.scala]: Great point, totally missed this, thanks Kapil.

File: test/uk/gov/hmrc/ups/scheduled/UpdatedPrintSuppressionJobSpec.scala (hunk 3 above)
Comment (ardwalker) [reply test/uk/gov/hmrc/ups/scheduled/UpdatedPrintSuppressionJobSpec.scala]: Many thanks for this @kjainsc

**Review by kjainsc — APPROVED** [review]

**Pull request merged**